### ISSUE TITLE
Timeseries DAO returning `TimeSeriesResults` when getting `Sites`

### ISSUE THREAD

**Issue by lsetiawan** [ODM2/WOFpy#116]
Based on @emiliom comment: https://github.com/ODM2/WOFpy/pull/114#discussion_r114661014

I opened up this issue as a reminder to revisit the [`odm2_timeseries_dao.py`](https://github.com/ODM2/WOFpy/blob/master/wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py) regarding what's queried, especially `get_all_sites`.

```python
def get_all_sites(self):
        s_rArr = self.db_session.query(odm2_models.Sites, odm2_models.TimeSeriesResults).\
            join(odm2_models.FeatureActions).\
            filter(odm2_models.FeatureActions.SamplingFeatureID == odm2_models.Sites.SamplingFeatureID,
                   odm2_models.TimeSeriesResults.FeatureActionID == odm2_models.FeatureActions.FeatureActionID). \
            group_by(odm2_models.Sites.SamplingFeatureID).all()
        ...
```

`TimeSeriesResults` are queried.. why? Our thoughts are to remove this query to speed up the process.

**Comment by emiliom** [comment]
> TimeSeriesResults are queried.. why? Our thoughts are to remove this query to speed up the process.

More specifically, having `TimeSeriesResults` in the `query` method only results in having `TimeSeriesResults` returned by the query. But AFAIK it's only needed as a query **criteria**, to select only Sites that are associated with `TimeSeriesResults`. So I don't see why it needs to be **returned** by the query.

Low priority ...

**Comment by emiliom** [comment]
Closing. Addressed in PR #126
